frontend/src/console/actions.ts

```diff
@@ -7,7 +7,11 @@ import {
   DeploymentInfoModel,
   DeploymentNotificationModel
 } from 'console/models';
-import { ThemeSchema, StaticContentOverrides } from 'ocim-client';
+import {
+  ThemeSchema,
+  StaticContentOverrides,
+  ToggleStaticContentPages
```

Comment:
I'm sorry in advance for all the trailing comma comments I'll be adding. :slightly_frowning_face: 

```suggestion
  ToggleStaticContentPages,
```

Reply:
@s0b0lev please let me know if TypeScript doesn't like trailing commas :+1:

Reply:
Yeh, pretier would complain for such trailing comma.
```
/home/s0b0lev/ocim/frontend/src/console/actions.ts
  13:27  error  Delete `,`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
Oh :slightly_frowning_face: 

Then ignore all typescript trailing commas :+1: Sorry for the inconvenience :slightly_frowning_face: